spec/services/batch_invite_job_spec.rb

```diff
@@ -49,6 +49,13 @@
          (org.id+1) => 'Error: Email has already been taken'}
       )
     end
+
+
+    it 'resend invitation to a deleted user' do
+      deleted_user = FactoryGirl.create(:user,email: 'YES@hello.com', deleted_at: "2015-11-11 08:37:14")
+      expect(-> { subject }).to change(User, :count).by(1)
```

Comment:
should re-run the invitation given that the delete took place, and make sure that the error doesn't occur